timestamps: Adding a correct serde instant converter for gson.

martin-bucinskas opened the pull request:
## Whats changed

- Gson converts instants to seconds and nanoseconds, this formats it to an ISO string.
- Externalising the `mockDcs` flag so that this can be triggered via config change rather than a code deploy.

martin-bucinskas committed:
timestamps: Adding a correct serde instant converter for gson.

- Gson converts instants to seconds and nanoseconds, this formats it to an ISO string.

kerrr commented on src/main/java/uk/gov/di/ipv/atp/dcs/utils/InstantConverter.java:
```diff
@@ -0,0 +1,23 @@
+package uk.gov.di.ipv.atp.dcs.utils;
+
+
+import com.google.gson.*;
```
We should avoid wildcard imports.

https://gds-way.cloudapps.digital/manuals/programming-languages/java.html#imports

martin-bucinskas replied:
Ah, I'll change the IDE settings 👍

kerrr replied:
🙇🏻‍♂️

martin-bucinskas committed:
timestamps: Not using wildcard imports.

- IntelliJ by default merges imports together with a wildcard if 5 or more imports from the same base package are imported.
